Updates for golden image heterogenous cluster support (#99811)
openshift/openshift-docs: modules/virt-creating-vm-from-template.adoc

** For a general view, navigate to *Virtualization* -> *Catalog*.
. Click the *Template catalog* tab.
. Click the *Boot source available* checkbox to filter templates with boot sources. The catalog displays the default templates.

> . Click the applicable options in the *Architecture Type* section to filter on templates associated with a particular architecture. This is only available in heterogeneous clusters.

Reviewer: ```suggestion
. Heterogeneous clusters only: To filter the search results to show templates associated with a particular architecture, click *Architecture Type* .
```

Author: Fixed. Thank you!
